pages/common/dot.md

```diff
@@ -1,12 +1,25 @@
 # dot
 
-> A command-line tool to produce layered drawings of directed graphs.
-> More information: <https://www.graphviz.org/pdf/dotguide.pdf>.
+> Render an image of a _linear directed_ network graph from a `graphviz` file.
```

Comment:
```suggestion
> Render an image of a `_linear directed_` network graph from a `graphviz` file.
```

Reply:
Yes, all of the layout commands are waiting on the resolution of this command to be added. See the attached autogen template. I have accepted the suggestion to use `linear directed` instead of _linear directed_ to highlight the specific type of layout provided by the command. See next commit.